Cargo.toml

```diff
@@ -1,8 +1,22 @@
 [workspace]
-
 members = [
   "rumqttc",
   "rumqttd",
   "benchmarks",
   "benchmarks/simplerouter",
 ]
+
+[workspace.package]
+rust-version = "1.70.0"
```

Comment:
according to docs[1] this is minimal supported rust version, and i don't think you need `1.70` to compile this project. lets just remove it?

1: https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

Reply:
I've set Rust 1.70 because a dependency of `rumqttd` requires it to compile (you can try this using a Dockerfile and specifying a rust version < 1.70).
Since `rumqttc` is a library and does not depend on `rumqttd`, the only change that needs to be made is to override `rust-version` just for `rumqttc`. Do you know what the minimum version is?

Reply:
Can we use this to find out the msrv? https://github.com/foresterre/cargo-msrv

Reply:
and as per docs quoted by @henil :

> The Rust version must be a bare version number with two or three components; it cannot include semver operators or pre-release identifiers. Compiler pre-release identifiers such as -nightly will be ignored while checking the Rust version. 

just in case it's the correct version, we can't specify it like `1.70.0`, we have to use `1.70` ( according to their example )

Reply:
Yeah nice! 
Awesome 😎

Reply:
@swanandx it says there can be 
> with two or three components

so `1.70.0` also works, and the next release is `1.71.0`

Reply:
btw, can we once verify it its the correct msrv using cargo-msrv ?

Reply:
yeah i verified with `cargo-msrv`